external/vcpkg_custom_data/toolchains/linux_32.cmake

```diff
@@ -0,0 +1,572 @@
+        ERROR_FILE "${CMAKE_BINARY_DIR}/vcpkg-manifest-install-err.log"
+        RESULT_VARIABLE _VCPKG_INSTALL_RESULT
+    )
+
+    if (NOT _VCPKG_INSTALL_RESULT EQUAL 0)
+        message(FATAL_ERROR "Running vcpkg install - failed")
+    endif()
+
+    message(STATUS "Running vcpkg install - done")
+
+    set_property(DIRECTORY APPEND PROPERTY CMAKE_CONFIGURE_DEPENDS
+        "${_VCPKG_MANIFEST_DIR}/vcpkg.json"
+        "${_VCPKG_INSTALLED_DIR}/vcpkg/status")
+endif()
+
+option(VCPKG_APPLOCAL_DEPS "Automatically copy dependencies into the output directory for executables." ON)
+function(add_executable name)
+    _add_executable(${ARGV})
+    list(FIND ARGV "IMPORTED" IMPORTED_IDX)
+    list(FIND ARGV "ALIAS" ALIAS_IDX)
+    list(FIND ARGV "MACOSX_BUNDLE" MACOSX_BUNDLE_IDX)
+    if(IMPORTED_IDX EQUAL -1 AND ALIAS_IDX EQUAL -1)
+        if(VCPKG_APPLOCAL_DEPS)
+            if(_VCPKG_TARGET_TRIPLET_PLAT MATCHES "windows|uwp")
+                add_custom_command(TARGET ${name} POST_BUILD
+                    COMMAND powershell -noprofile -executionpolicy Bypass -file ${_VCPKG_TOOLCHAIN_DIR}/msbuild/applocal.ps1
+                        -targetBinary $<TARGET_FILE:${name}>
+                        -installedDir "${_VCPKG_INSTALLED_DIR}/${VCPKG_TARGET_TRIPLET}$<$<CONFIG:Debug>:/debug>/bin"
+                        -OutVariable out
+                )
+            elseif(_VCPKG_TARGET_TRIPLET_PLAT MATCHES "osx")
+                if (NOT MACOSX_BUNDLE_IDX EQUAL -1)
+                    add_custom_command(TARGET ${name} POST_BUILD
+                    COMMAND python ${_VCPKG_TOOLCHAIN_DIR}/osx/applocal.py
+                        $<TARGET_FILE:${name}>
+                        "${_VCPKG_INSTALLED_DIR}/${VCPKG_TARGET_TRIPLET}$<$<CONFIG:Debug>:/debug>"
+                    )
+                endif()
+            endif()
+        endif()
+        set_target_properties(${name} PROPERTIES VS_USER_PROPS do_not_import_user.props)
+        set_target_properties(${name} PROPERTIES VS_GLOBAL_VcpkgEnabled false)
+    endif()
+endfunction()
+
+function(add_library name)
+    _add_library(${ARGV})
+    list(FIND ARGV "IMPORTED" IMPORTED_IDX)
+    list(FIND ARGV "INTERFACE" INTERFACE_IDX)
+    list(FIND ARGV "ALIAS" ALIAS_IDX)
+    if(IMPORTED_IDX EQUAL -1 AND INTERFACE_IDX EQUAL -1 AND ALIAS_IDX EQUAL -1)
+        get_target_property(IS_LIBRARY_SHARED ${name} TYPE)
+        if(VCPKG_APPLOCAL_DEPS AND _VCPKG_TARGET_TRIPLET_PLAT MATCHES "windows|uwp" AND (IS_LIBRARY_SHARED STREQUAL "SHARED_LIBRARY" OR IS_LIBRARY_SHARED STREQUAL "MODULE_LIBRARY"))
+            add_custom_command(TARGET ${name} POST_BUILD
+                COMMAND powershell -noprofile -executionpolicy Bypass -file ${_VCPKG_TOOLCHAIN_DIR}/msbuild/applocal.ps1
+                    -targetBinary $<TARGET_FILE:${name}>
+                    -installedDir "${_VCPKG_INSTALLED_DIR}/${VCPKG_TARGET_TRIPLET}$<$<CONFIG:Debug>:/debug>/bin"
+                    -OutVariable out
```

Comment:
and here.  I only ask becuase this seems against the convention of the other switches.